security_monkey/sso/views.py

```diff
@@ -291,6 +294,9 @@ def post(self):
         r = requests.get(people_api_url, headers=headers)
         profile = r.json()
 
+        if 'email' not in profile:
+            raise Exception("Unable to fetch user e-mail. Please ensure your application has Google+ API access")
```

Comment:
Would prefer a more specific exception class.

Reply:
For both cases, or just for this exception?

Reply:
Both

Reply:
LGTM